hack/run-integration-tests.sh

```diff
@@ -31,19 +31,6 @@ EOF
   exit 0
 fi
 
-install_kubectl_if_needed() {
```

Comment:
This function is still referenced in https://github.com/kubernetes-sigs/krew/blob/e643ab39cfacdf6f30943bcc4f27d65cbf142553/hack/run-integration-tests.sh#L41
That's also why travis failed.